ios/SelectTokenIntentUI/IntentViewController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  IntentViewController.swift
+//  SelectTokenIntentUI
+//
+//  Created by Ben Goldberg on 10/28/21.
+//
+
+import IntentsUI
+
+// As an example, this extension's Info.plist has been configured to handle interactions for INSendMessageIntent.
```

Comment:
this file looks messy and those comments look useless? Can you lint and remove those comments? Do we actually need those files?

Reply:
Not needed. Removed